src/test/kotlin/no/nav/familie/tilbake/behandling/BehandlingRepositoryTest.kt

```diff
@@ -24,22 +25,22 @@ internal class BehandlingRepositoryTest : OppslagSpringRunnerTest() {
     }
 
     @Test
-    fun insertPersistererEnForekomstAvBehandlingTilBasen() {
+    fun `skal_persistere_en_behandling`() {
```

Comment:
Som jeg kommenterte tidligere når du spurte. God testnavning er navnet på metoden du tester og hva den skal gjøre. 
Jeg tester at metoden "insert" persitsterer en forekomst i basen. Hvis du ikke liker ordlyden og camel-casing i tester, så bør det i så fal være `insert skal persistere en behandling`. Snake-casing i testnavn brukes ikke i Kotlin.

Reply:
Jeg sjekket både familie-ba-sak og familie-ef-sak. Ingen har fulgt å bruke testnavn på den måten. Jeg tror vi bør være konsistens. Jeg fulgte å bruke ef-sak testnavn 'skal opprette automatisk behandling uten verge`, fjernet underscore.

Reply:
Problemet med å ikke si hvilken metode du tester er at når en test feiler, så må du lese testkoden for å vite hva som blir testet. Det tar dermed lengre tid å finne og fikse feilen når en test brekker. 
Ref: 
https://osherove.com/blog/2005/4/3/naming-standards-for-unit-tests.html
https://qualitycoding.org/unit-test-naming/